app/components/admin/worldwide_organisation_pages/index/summary_card_component.rb

```diff
@@ -1,14 +1,21 @@
 # frozen_string_literal: true
 
 class Admin::WorldwideOrganisationPages::Index::SummaryCardComponent < ViewComponent::Base
-  attr_reader :page
+  attr_reader :page, :worldwide_organisation
 
-  def initialize(page:)
+  def initialize(page:, worldwide_organisation:)
     @page = page
+    @worldwide_organisation = worldwide_organisation
   end
 
 private
 
+  def title
+    I18n.with_locale(:en) do
```

Comment:
Sorry, just got some email notifications from this PR and noticed I didn't self-review it very well. I don't think this block is needed, the tests should catch whether it is

Reply:
Thanks, I've removed this block and confirmed it isn't needed.